smalld/__init__.py

```diff
@@ -1 +1,3 @@
+from requests.exceptions import *  # noqa
```

Comment:
I'm not super excited about a * here, especially from a 3rd party package. I'd like to better understand what exceptions we might be throwing.

Even better, would be exporting the exceptions in the smalld namespace, meaning users of the lib won't have to be importing form requests. Also useful if we decide to switch from requests.

Reply:
At first I saw there were like 20 exceptions and didn't fancy the idea of importing them explicitly, but I think not all of them should be exported anyways :thinking:.

That's what I am doing by importing the exceptions here, users can just `import HTTPError from smalld`. Unless you mean to make our own exceptions, I prefer exporting the requests exceptions as is.

Reply:
ah right, importing them here would already put them in the smalld name space. I was thinking we'd have to do HttpError = requests.exceptions.HttpError, not sure why I thought that.

I think if we can get a list instead of * that would be good. We probably need exceptions for http errors, connection errors, and rate limit?